root/static/scripts/common/components/DynamicAttributesList.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import bracketed from '../utility/bracketed';
+
+const DynamicAttributesList = ({entity}: {entity: WorkT}) => {
```

Comment:
Named it like this to already make it a bit more general in preparation for @yvanzo's work on attributes, which I imagine will lead to this component being used more widely :)

Reply:
Makes sense. The `lp_attributes` contexts are hardcoded for works, but I guess it can be figured out how to generalize that further when more entities are added.

Reply:
Simply name it `AttributesList` then, thanks!